CHANGELOG.md

```diff
@@ -6,10 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### Added
+- Fixed `_full_trfresh` method in `Databricks` source.
```

Comment:
1) typo, 2) don't add internal methods to changelog

Reply:
yes but still add the change :D I guess sth like "Fixed `Databricks.create_table_from_pandas()` failing to overwrite a table in some cases even with `replace="True"`"?

Reply:
Also you can always describe the change briefly and link the issue for more details, like `Fixed an issue with overriding tables in `Databricks.create_table_from_pandas()`, see  #issue_no`

Reply:
[📝 Added entry to CHANGELOG.md](https://github.com/dyvenia/viadot/pull/658/commits/0a6cac43e19dc64421a40a6462a7af5cd046b69f)